src/components/accounts/SetupSeedPhrase.js

```diff
@@ -166,6 +176,7 @@ class SetupSeedPhrase extends Component {
                                             formLoader={this.props.formLoader}
                                             requestStatus={this.state.requestStatus}
                                             globalAlert={this.props.globalAlert}
+                                            submitting={this.state.submitting}
```

Comment:
@Patrick1904 should few just pass `formLoader={this.props.formLoader || this.state.submitting}` here? Basically seems to me that this only needs to be handled once upstream vs passing downstream everwhere.